src/main/java/io/github/railroad/utility/localization/Languages.java

```diff
@@ -0,0 +1,44 @@
+package io.github.railroad.utility.localization;
+
+public enum Languages {
+    ENGLISH("en_us", "English"),
+    SPANISH("es_es", "Español"),
+    FRENCH("fr_fr", "Français"),
+    GERMAN("de_de", "Deutsch");
+
+    private final String locale;
+    private final String language;
+
+    Languages(String locale, String language) {
+        this.locale = locale;
+        this.language = language;
+    }
+
+    public String getLocale() {
+        return locale;
+    }
+
+    public String getLanguage() {
+        return language;
+    }
+
+    public static Languages fromLocale(String locale) {
+        for (Languages lang : values()) {
+            if (lang.getLocale().equals(locale)) {
+                return lang;
+            }
+        }
+
+        return null;
+    }
+
+    public static Languages fromLanguage(String language) {
+        for (Languages lang : values()) {
+            if (lang.getLanguage().equals(language)) {
+                return lang;
+            }
+        }
+
+        return null;
+    }
+}
```

Comment:
We could possibly consider just using `Locale` instead of this.

Reply:
I'm going to keep it using Languages for this PR, or atleast at the moment, I might move it over later but I'll prioritize updating all the components first.